sympy/solvers/ode/tests/test_systems.py

```diff
@@ -1040,10 +1040,166 @@ def test_sysode_linear_neq_order1_type2():
+                *Integral(-23*exp(-t)*sin(sqrt(2)*t) + 81*sqrt(2)*exp(-t)*cos(sqrt(2)*t), t)
+                + exp(t)*cos(sqrt(2)*t)*Integral(-23*exp(-t)*sin(sqrt(2)*t)**2/cos(sqrt(2)*t)
+                + 81*sqrt(2)*exp(-t)*sin(sqrt(2)*t) + 23*exp(-t)/cos(sqrt(2)*t), t))]
+    assert dsolve(eq12) == sol12
+    assert checksysodesol(eq12, sol12) == (True, [0, 0])
 
-    f, g, h, k = symbols('f g h k', cls=Function)
+    eq13 = [Eq(diff(f(t), t), f(t) + 2*g(t) + k1),
+            Eq(diff(g(t), t), 3*f(t) + 4*g(t) + k2)]
+    sol13 = [Eq(f(t),
+                -C1*exp(5*t/2 + sqrt(33)*t/2)/2 + sqrt(33)*C1*exp(5*t/2 + sqrt(33)*t/2)/6 - sqrt(33)*C2*exp(
+                -sqrt(33)*t/2 + 5*t/2)/6 - C2*exp(-sqrt(33)*t/2 + 5*t/2)/2 - exp(5*t/2 + sqrt(33)*t/2)*Integral(sqrt(
+                33)*k1*exp(-sqrt(33)*t/2 - 5*t/2)/11 + sqrt(33)*k2*exp(-sqrt(33)*t/2 - 5*t/2)/22 + k2*exp(-sqrt(
+                33)*t/2 - 5*t/2)/2, t)/2 + sqrt(33)*exp(5*t/2 + sqrt(33)*t/2)*Integral(sqrt(33)*k1*exp(-sqrt(
+                33)*t/2 - 5*t/2)/11 + sqrt(33)*k2*exp(-sqrt(33)*t/2 - 5*t/2)/22 + k2*exp(-sqrt(33)*t/2 - 5*t/2)/2, t)/6 - sqrt(
+                33)*exp(-sqrt(33)*t/2 + 5*t/2)*Integral(-sqrt(33)*k1*exp(-5*t/2 + sqrt(33)*t/2)/11 - sqrt(33)*k2*exp(
+                -5*t/2 + sqrt(33)*t/2)/22 + k2*exp(-5*t/2 + sqrt(33)*t/2)/2, t)/6 - exp(-sqrt(33)*t/2 + 5*t/2)*Integral(
+                -sqrt(33)*k1*exp(-5*t/2 + sqrt(33)*t/2)/11 - sqrt(33)*k2*exp(-5*t/2 + sqrt(33)*t/2)/22 + k2*exp(
+                -5*t/2 + sqrt(33)*t/2)/2, t)/2),
+             Eq(g(t),
+                C1*exp(5*t/2 + sqrt(33)*t/2) + C2*exp(-sqrt(33)*t/2 + 5*t/2) + exp(
+                5*t/2 + sqrt(33)*t/2)*Integral(sqrt(33)*k1*exp(-sqrt(33)*t/2 - 5*t/2)/11 + sqrt(33)*k2*exp(-sqrt(
+                33)*t/2 - 5*t/2)/22 + k2*exp(-sqrt(33)*t/2 - 5*t/2)/2, t) + exp(-sqrt(33)*t/2 + 5*t/2)*Integral(-sqrt(33)*k1*exp(
+                -5*t/2 + sqrt(33)*t/2)/11 - sqrt(33)*k2*exp(-5*t/2 + sqrt(33)*t/2)/22 + k2*exp(-5*t/2 + sqrt(33)*t/2)/2, t))]
+    assert dsolve(eq13) == sol13
+    assert checksysodesol(eq13, sol13) == (True, [0, 0])
+
+    eq14 = [Eq(diff(f(t), t), k1),
+            Eq(diff(g(t), t), k2)]
+    sol14 = [Eq(f(t), C1 + Integral(k1, t)),
+             Eq(g(t), C2 + Integral(k2, t))]
+    assert dsolve(eq14) == sol14
+    assert checksysodesol(eq14, sol14) == (True, [0, 0])
+
+    eq15 = [Eq(diff(f(t), t), 0),
+            Eq(diff(g(t), t), c*f(t) + k2)]
+    sol15 = [Eq(f(t), C1 + Integral(0, t)),
+             Eq(g(t), C1*c*t + C2*c + c*t*Integral(
+                 0, t) + c*Integral(k2/c, t))]
+    assert dsolve(eq15) == sol15
+    assert checksysodesol(eq15, sol15) == (True, [0, 0])
+
+    eq16 = [Eq(diff(f(t), t), k1),
+            Eq(diff(g(t), t), f(t) + k2)]
+    sol16 = [Eq(f(t), C1 + Integral(k1, t)),
+             Eq(g(t), C1*t + C2 +
+                t*Integral(k1, t) +
+                Integral(-k1*t + k2, t))]
```

Comment:
These three lines would all fit neatly on one line and would then be easier to read. (There are other examples of this)

Reply:
have updated the formatting of the shorter equations